config/jobs/kubernetes-sigs/descheduler/descheduler-presubmits-release-1.24.yaml

```diff
@@ -0,0 +1,145 @@
+# sigs.k8s.io/descheduler presubmits
+presubmits:
+  kubernetes-sigs/descheduler:
+  - name: pull-descheduler-verify-release-1-24
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: pull-descheduler-verify-release-1.24
+    decorate: true
+    path_alias: sigs.k8s.io/descheduler
+    branches:
+    # The script this job runs is not in all branches.
+    - ^release-1.23$
```

Comment:
this should be `release-1.24`

Reply:
Thanks for pointing this out, I've updated it with latest push.